templates/BuildMyCharm/config.xml

```diff
@@ -84,7 +103,7 @@ fi
     <org.jenkinsci.plugins.postbuildscript.PostBuildScript plugin="postbuildscript@0.17">
       <buildSteps>
         <hudson.tasks.Shell>
-          <command>juju destroy-model test-{{charmname}}-$BUILD_NUMBER -y</command>
+          <command>juju destroy-model test-{{charmname}}-$BUILD_NUMBER -y || true</command>
```

Comment:
I am a bit sceptical about this. I know it makes the first dummy build green. However, it also hides any cleanup operation that fails. In my mind it is more important to know that the cloud resources did not get released (as I might be paying for this).